project/src/test/java/com/google/sps/data/PlacesFetcherTest.java

```diff
@@ -47,7 +47,7 @@
   private static final float RATING = 4;
   private static final int PRICELEVEL_INT = 2; // used for Places and UserPreferences
   private static final PriceLevel PRICELEVEL = PriceLevel.MODERATE; // used for PlaceDetails
-  private static final ImmutableList<String> CUISINES = ImmutableList.of("sushi", "burger");
+  private static final ImmutableList<String> CUISINES = ImmutableList.of("sushi", "hamburger");
```

Comment:
Missing a test for validating the configuration's use. Something like making sure that the string that was used in the query contained what it is that you want.

Reply:
I added tests. It made me change the visibility of the createCuisinesQuery() function so let me know what you think about it.
Also adding the tests reminded me that I didn't handle invalid cuisines input so I added an exception throw for that case.